CMakeLists.txt

```
@@ -157,9 +157,11 @@ endif()
 
 set(CMAKE_C_FLAGS_DEBUG "${C_debug} ${C_checked}")
 set(CMAKE_C_FLAGS_RELEASE "${C_optimized}")
+set(CMAKE_C_FLAGS_RELWITHDEBINFO "${C_debug} ${C_optimized}")
```

For consistency, let's swap it around:
```suggestion
set(CMAKE_C_FLAGS_RELWITHDEBINFO "${C_optimized} ${C_debug}")
```